app/config/parameters.yml.dist

```diff
@@ -53,3 +53,6 @@ parameters:
 
     applicolis_api_key: ~
     applicolis_api_base_url: ~
+
+    # For the guzzle client
+    osrm_url: 'http://osrm:5000'
```

Comment:
There is already the `osrm_host` parameter, do we really need another parameter?